Src/PerformanceCollector/Shared/Implementation/QuickPulse/Process/IQuickPulseTopCpuCollector.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.QuickPulse
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Interface for top CPU collector.
+    /// </summary>
+    internal interface IQuickPulseTopCpuCollector
+    {
+        bool InitializationFailed { get; }
+
+        bool AccessDenied { get; }
+
+        IEnumerable<Tuple<string, int>> GetTopProcessesByCpu(int topN);
```

Comment:
There's a school of thought that I have learned to believe in from experience that says document your code so other people can read the documentation.  Have you ever tried to read code in Reflector that has no comments?  It's not fun.  Have you ever tried to look something up on MSDN and found it had no comments/documentation?  Not fun either.